armi/reactor/converters/blockConverters.py

```diff
@@ -645,6 +661,25 @@ def _classifyComponents(self):
 
         return list(sorted(pinComponents)), nonPins
 
+    def _buildInsideDuct(self):
+        """Build a homogenized material of the components inside the duct."""
+        blockType = self._sourceBlock.getType()
+        blockName = f"Homogenized {blockType}"
+        newBlock, mixtureFlags = stripComponents(self._sourceBlock, Flags.DUCT)
+        outerDiam = getOuterDiamFromIDAndArea(0.0, newBlock.getArea())
+        circle = components.Circle(
```

Comment:
Is this really supposed to be a circle? Or should it be derived from the shape of the duct, i.e. hexagon?

Reply:
Removing this one since I was being dumb.